pkg/clustermanager/wireguard.go

```diff
@@ -82,3 +82,23 @@ func GenerateKeyPair() (WgKeyPair, error) {
 		Public:  base64.StdEncoding.EncodeToString(publicKey[:]),
 	}, nil
 }
+
+//GenerateEtcdSystemdService generate configuration file used to manage etcd service on systemd
+func GenerateOverlayRouteSystemdService(node Node) string {
+	serviceTpls := `# /etc/systemd/system/overlay-route.service
+[Unit]
+Description=Overlay network route for Wireguard
+After=wg-quick@wg0.service
+
+[Service]
+Type=oneshot
+User=root
+ExecStart=/sbin/ip route add 10.96.0.0/12 dev wg0 src %s
```

Comment:
is that for routing pod networking over wireguard? Did you check that

Reply:
That one is for Weave Net to make sure it routes traffic via the correct network interface. More on it here https://github.com/hobby-kube/guide

Reply:
If we do this for canal, maybe we should check for the correct IP range.

Reply:
added!